pybossa/cache/projects.py

```diff
@@ -105,7 +105,7 @@ def format_task(row, lock_users=[]):
     # avoid sql planer doing indexscan when searching through table,
     # the settings is LOCAL and only applies to current transaction,
     # This optimization is aiming to resolve Task Browse Page slowness for those take >2s to complete.
-    # TODO: investigate cache to further improve performance of Task List page
+    # TODO: upgrade PostgreSQL from v10 to v12 to check if SQL planner does a better job on Task Browse
```

Comment:
I have my local with v13 with 10M records in both task and task_run tables and it doesn't seem to give me a good result for the original task browse SQL. I think the way you proposed to optimize using CTE is the way to go. Feel free to test it out in your local env.

Reply:
The comments [here](https://github.com/bloomberg/pybossa/pull/697#discussion_r814928774) are obsolete now and can be removed.

Reply:
what comment are you referring to? "upgrade PostgreSQL from v10 to v12 to check if SQL planner does a better job on Task Browse"?

Reply:
ignore incorrect link; added my comment to the correct part